clients/src/main/java/org/apache/kafka/clients/admin/KafkaAdminClient.java

```diff
@@ -690,6 +690,13 @@ public Node provide() {
             this(false, callName, deadlineMs, nodeProvider);
         }
 
+        Call(String callName, long deadlineMs, NodeProvider nodeProvider,
+            Optional<Integer> numTries, Optional<Long> nextAllowedTryMs) {
```

Comment:
Not sure if we need to make these `Optional`. `0` seems to be a good default value for these.

Reply:
I initially thought that we might need one, because we wouldn't know, but after thinking about your comment that `0` is a good default value, I have changed it to reflect as such.